### ISSUE TITLE
File::stat::stat throws an error on Microsoft Windows Server 2012 R2 Standard

### ISSUE THREAD

**Issue by aibistin** [dagolden/Path-Tiny#151]
I'm not sure if this is a Path::Tiny issue or a Strawberry Perl issue. The server is running Strawberry Perl(perl 5, version 22, subversion 0 (v5.22.0) built for MSWin32-x64-multi-thread).
The Path::Tiny version is '0.072'.

This is the error message:
,Undefined subroutine &File::stat::stat called at C:/Strawberry/perl/site/lib/Path/Tiny.pm line 1542.

I tested File::stat using the exported 'stat' function and it seems to work fine. Any help would be appreciated.
Thanks.

**Comment by dagolden** [comment]
The 'stat' method gets exercised in the test suite and there are no Win32 failures from CPAN Testers.

Did this error come from the test suite or from some program you have that's using Path::Tiny?

**Comment by aibistin** [comment]
Actually this is coming from a script that I have that uses Path::Tiny. I just found a fix for this. The script ran fine in Strawberry perl 5.18. The problem was with 5.22. I noticed that this version also installs 'File::Stat' by default. Once I removed 'File::Stat' from its directory (C:\Strawberry\perl\lib\File\Stat.pm), the problem disappeared. Somehow File::Stat was being called instead of File::stat ?? That's just my guess. 
Thanks for your quick response. And thanks for Path::Tiny.
